modules/ocf_mirrors/manifests/init.pp

```diff
@@ -45,6 +47,10 @@
       source => 'puppet:///modules/ocf_mirrors/README.html',
       owner  => mirrors,
       group  => mirrors;
+
+    '/usr/local/sbin/record-mirrors-stats':
+      content => template('ocf_mirrors/record-mirrors-stats.py.erb'),
+      mode    => '0644';
```

Comment:
1. I don't think this should be world-readable to be consistent with our use of secrets elsewhere.
2. Shouldn't this be executable?